composer.json

```diff
@@ -34,9 +36,18 @@
         }
     },
     "require": {
-        "laravel/passport": "^10|^11",
         "laravel/legacy-factories": "^1.3",
-        "nl.idaas/openid-server": "^0.4.2"
+        "nl.idaas/openid-server": "^0.4.2",
+        "illuminate/auth": "^10.0|^11.0",
+        "illuminate/console": "^10.0|^11.0",
+        "illuminate/container": "^10.0|^11.0",
+        "illuminate/contracts": "^10.0|^11.0",
+        "illuminate/cookie": "^10.0|^11.0",
+        "illuminate/database": "^10.0|^11.0",
+        "illuminate/encryption": "^10.0|^11.0",
+        "illuminate/http": "^10.0|^11.0",
+        "illuminate/support": "^10.0|^11.0",
+        "laravel/passport": "^10.0|^11.0"
```

Comment:
I ran the tests without these and it passes. I realized because laravel/passport already includes them as dependency.

Do you intend to force minimum Laravel to be Laravel 10 and 11?